src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -3,21 +3,24 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import javax.annotation.Nullable;
+
 import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
 import teammates.common.util.Const;
 
 public class FeedbackTextQuestionDetails extends FeedbackQuestionDetails {
 
-    private int recommendedLength;
+    @Nullable
+    private Integer recommendedLength;
```

Comment:
Just a note. This will change the things stored in database (should be backward-compatible though)

Reply:
I added this line `if (recommendedLength != null && recommendedLength < 1)` which would make all forms using the previous default value show an error if they tried to save. Would this be an issue?

Reply:
This is fine as the frontend is bundled with the backend in production.

Reply:
This should be backward compatible because it's equivalent to removing a field. If it's adding a field then we need to worry.